### DISCUSSION TITLE
https://plugin-search-pro.vuejs.press/zh/config.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#610] [Announcements]
# https://plugin-search-pro.vuejs.press/zh/config.html

插件选项 indexContent 类型: boolean; 默认值: false; 是否索引内容。 默认情况下，插件只会索引页面的标题和摘要以及你的自定义索引项，不会索引页面的正文内容。如果需要索引页面的正文内容，可以将该选项设置为 true。 autoSuggestions 类型: boolean; 默认值: false; 是否自动提示搜索建议。 ...

https://plugin-search-pro.vuejs.press/zh/config.html

**Comment by sunxf94** [comment]
delay 选项已经更新为 searchDelay，辛苦Hope大大更新文档

**Reply by Mister-Hope** [reply]
已经更新，感谢反馈。

**Comment by OrageKK** [comment]
page.frontmatter.category 和page.frontmatter.tag 已经不在frontmatter选项中，希望更新一下文档

**Reply by Mister-Hope** [reply]
你说的两个东西是 vuepress-theme-hope 或者其他主题特有的，插件并不清楚 VuePress 内置键名之外的任意键会被你设成什么东西

**Comment by vipdcd** [comment]
开发环境正常，但build之后报以下错误：“Uncaught SyntaxError: Unexpected token '<' (at search-pro.worker.js:1:1)”，麻烦帮指点下如何解决，谢谢！

**Reply by Mister-Hope** [reply]
提供一个最小复现并开一个issue

**Comment by qinxin09** [comment]
输入时抖动得厉害，可以考虑设个高度
